src/components/NewExperiment.vue

```diff
@@ -26,7 +26,7 @@
                   <v-data-table
                     :headers="selectedTable.headers"
                     :items="tables.conditions.items"
-                    :pagination.sync="pagination"
```

Comment:
I would keep something like:
```
:pagination.sync="disabledToPreventWrongIndexOnSecondPage"

// We are relying on data table's `index` (for currentRowIndex and @paste)
// but that only works correctly on the first page.
disabledToPreventWrongIndexOnSecondPage: undefined
```
to avoid future bugs